src/github.js

```diff
@@ -359,6 +359,15 @@ export default function githubFactory(token, repository) {
                 })
                 .map(this.formatReleaseNote)
                 .reduce((acc, note) => note ? `${acc} - ${note}\n` : acc, '');
+        },
+
+        /**
+         * Search for pull requests using GitHub GraphQL API
+         * @param {String} searchQuery - GitHub search query
+         * @returns {Promise<Object>} resolves with search results
+         */
+        searchPullRequests(searchQuery) {
```

Comment:
I think its unused? [Here](https://github.com/oat-sa/tao-extension-release/pull/140/files#diff-5434bdbe127d5cabdb2500c732aa230a2ae318dc0eba5adba6da8b2f3b181afdR284) you just use `githubApiClient.searchPullRequests(searchQuery)`